src/app/application.cpp

```diff
@@ -557,76 +560,123 @@ void Application::processParams(const QStringList &params)
 
 int Application::exec(const QStringList &params)
 {
+#ifndef DISABLE_GUI
+    QPixmap splashImg {":/icons/splash.png"};
```

Comment:
The old !(params.noSplash || Preferences::instance()->isSplashScreenDisabled()) check is removed entirely, so splash screen is displayed regardless of preferences, as xavier mentioned.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/13498#issuecomment-706839330